lib/telemetry_metrics_statsd/formatter/datadog.ex

```diff
@@ -25,8 +25,15 @@ defmodule TelemetryMetricsStatsd.Formatter.Datadog do
 
   defp format_metric_value(%Metrics.Counter{}, _value), do: "1|c"
 
-  defp format_metric_value(%Metrics.Summary{}, value),
-    do: [format_number(value), "|ms"]
+  defp format_metric_value(
+         %Metrics.Summary{reporter_options: reporter_options},
```

Comment:
@hkrutzer how about we make `Distribution` reportable in Datadog format?

Reply:
The types of telemetry_metrics vs statsd metrics is a bit confusing, but the telemetry_metrics type requires a `buckets` option right? That doesn't make much sense here.. But the name would make a lot more sense.

Reply:
Yes, that doesn't make much sense. I'll bring that up in Telemetry.Metrics.

Reply:
https://github.com/beam-telemetry/telemetry_metrics/issues/60